instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/otel/OtelJvmGcMetrics.java

```diff
@@ -98,24 +85,13 @@ public void install() {
 
     Meter meter = OtelMeterProvider.get();
 
-    GcMetricsNotificationListener gcNotificationListener = new GcMetricsNotificationListener(meter);
-
-    double maxLongLivedPoolBytes =
-        getLongLivedHeapPools().mapToDouble(mem -> getUsageValue(mem, MemoryUsage::getMax)).sum();
+    GcMetricsNotificationListener gcNotificationListener = new GcMetricsNotificationListener();
 
-    maxDataSize = new AtomicLong((long) maxLongLivedPoolBytes);
-    meter
-        .gaugeBuilder("runtime.jvm.gc.max.data.size")
-        .setUnit("bytes")
-        .setDescription("Max size of long-lived heap memory pool.")
-        .buildWithCallback(measurement -> measurement.record(maxDataSize.get()));
+    // runtime.jvm.gc.max.data.size is replaced by OTel
+    //   process.runtime.jvm.memory.limit{pool=<long lived pools>}
 
-    liveDataSize = new AtomicLong();
-    meter
-        .gaugeBuilder("runtime.jvm.gc.live.data.size")
-        .setUnit("bytes")
-        .setDescription("Size of long-lived heap memory pool after reclamation.")
-        .buildWithCallback(measurement -> measurement.record(liveDataSize.get()));
+    // runtime.jvm.gc.live.data.size is replaced by OTel
+    //   process.runtime.jvm.memory.usage_after_last_gc{pool=<long lived pools>}
 
```

Comment:
There are two other metrics left in this class: `runtime.jvm.gc.memory.allocated` and `runtime.jvm.gc.memory.promoted`. Both of these emit data that needs to be calculated on the instrumentation side, I think it's probably not possible to reproduce these by querying the backend (dashboards).

Still, I think we should remove them: both of them depend on hard-coded GC & memory pool names, long-term maintaining them and ensuring they're correct for all GC possible implementations is going to be painful.
Also I don't know how useful these really are, maybe the "standard" memory & GC metrics are enough anyway?

Reply:
Ok good with that. It does seem weird that we have `memory.allocated` inside a `runtime.jvm.gc` namespace anyway. 

I'm not sure if you saw the JFR based approach in contrib...but it's also nontrivial and bakes in some expectations about names.